src/js/view/view.js

```diff
@@ -57,7 +57,11 @@ function View(_api, _model) {
     });
 
     const _localization = _model.get('localization');
-    const _playerElement = createElement(playerTemplate(_model.get('id'), _localization.player));
+    const _playerElement = createElement(playerTemplate(
+        _model.get('id'), 
+        _localization.player.label, 
```

Comment:
player is not an object, this might be why your commercial tests fail

Reply:
Good catch, I missed this when I moved the copy out of localization